receiver/sqlserverreceiver/documentation.md

```diff
@@ -242,6 +242,25 @@ This metric is only available when the receiver is configured to directly connec
 | ---- | ----------- | ------ |
 | database.status | The current status of a database | Str: ``online``, ``restoring``, ``recovering``, ``pending_recovery``, ``suspect``, ``offline`` |
 
+### sqlserver.database.io
```

Comment:
`sqlserver.database.io` is used as a namespace in `sqlserver.database.io.write_latency`, so it cannot be used as metric name according to the OTel spec. Probably `sqlserver.database.io.bytes`?

Reply:
I believe this is no longer a concern since the metric `sqlserver.database.io.write_latency` has been renamed to `sqlserver.database.latency`, but let me know if I misunderstood.